### DISCUSSION TITLE
Setting alias via CLI is returning 508

### DISCUSSION THREAD

**Discussion by evelynstender** [vercel/vercel#8154] [Help] [answered]
Hi, I'm trying to manually set an alias via vercel CLI, but I'm getting an 508 error back. I tried running like this:

`vercel alias set preview-url.com mydomain.com --scope my-scope --token my-token` and I'm getting this error back

```bash
Vercel CLI 25.1.0
> Assigning alias mydomain.com to deployment preview-url.com
Fetching domain mydomain.com under my-scope
Error! An unexpected error occurred in alias: Error: Response Error (508)
```

Anyone knows what this could be?

Thank you

**Comment by evelynstender** [comment]
This also happens using `Vercel CLI 27.1.2`

**Comment by amyegan** [comment]
Hi @evelynstender. Thanks for trying with the latest version of the CLI, and sorry to hear it still isn't working for you. In the Project settings, can you see if your custom domain is set to redirect or forward to another address? You also might not need to set `--scope` and `--token` if you have [linked your project with Vercel CLI](https://vercel.com/docs/cli#introduction/project-linking)

**Comment by evelynstender** [comment, marked as answer]
I figured out the issue.

The problem is I was trying to manually assign the domain, however the domain was already setup to be assigned automatically via branch name. I just stopped trying to assign manually and let Vercel do the magic! 

Thank you @amyegan for your time!

**Reply by amyegan** [reply]
Thanks for coming back to share the answer!

**Comment by andrewdavid48** [comment]
@amyegan what about in a CI pipeline? I need to set the alias for a custom preview environment via github actions, and everything I've tried returns 508. But locally the alias works perfectly... This is concerning and is making my team move away from vercel.
